contracts/contracts/interfaces/IVault.sol

```diff
@@ -165,9 +165,9 @@ interface IVault {
 
     function netOusdMintForStrategyThreshold() external view returns (uint256);
 
-    function setOusdMetaStrategy(address _ousdMetaStrategy) external;
+    function setMetaStrategy(address _strategy) external;
```

Comment:
We don't plan to use this for OETH though right? This is exclusive to OUSDMetStrategy contract. the curve AMO uses mintForStrategy I believe

Reply:
It currently is on the OETHVaultAdmin hence the reason for are trying to clean it up.

There may be a better way of doing this. eg
1. Move these OUSD specific functions to an OUSD Vault implementation contract
2. Change the OUSD strategies to use the more generic functions.

Both sound like a bigger change for another day.